lib/ravelin/event.rb

```diff
@@ -56,8 +60,6 @@ def validate_top_level_payload_params
           :payment_method_id, :payment_method
         )
         validate_payload_inclusion_of :order_id
-      when :login
-        validate_payload_inclusion_of :customer_id
```

Comment:
Why remove this?

Reply:
The majority of the code in this repo is about forming JSON payloads based on simple classes. It rewrites a lot of names, and tries to do clever serialization and deserialization where it interprets strings to names of classes, etc.

That's fine, but the problem is sometimes names are used twice in the ravelin request. `Login` is kind of used twice, because it's the name of the endpoint, but it's also the name of a parent in the JSON, so it ends up trying to validate the inclusion twice - but the real `Login` object has the `attr_required` for `customer_id` anyway